server/app/views/admin/programs/ProgramBlocksView.java

```diff
@@ -434,7 +435,8 @@ private DivTag renderBlockPanel(
       ButtonTag blockDeleteModalButton,
       boolean isIntakeFormFeatureEnabled,
       Request request) {
-    // A block can only be deleted when it has no repeated blocks. Same is true for removing the
+    // A block can only be deleted when it has no repeated blocks. Same is true for
+    // removing the
```

Comment:
nitpick: I guess `bin/fmt` decided to put the line breaks in different places 😂 will you move the next comment line (`// enumerator question from the block.`) up to this one so we don't have this weird two-word comment line? It looks like there are a bunch of places below where comments got split up in a strange way as well. Please fix those too.